examples/mobile/HomeApp/webclip/restaurant/hotel-breakfast-wc.js

```diff
@@ -1,107 +1,6 @@
 /* global tau */
-const tmpl = document.createElement("template");
```

Comment:
Is there any reason to move it to html?
Is it necessary to switch pages in order for webclip to work?